src/Components/Auction/PostRegistrationModal.tsx

```diff
@@ -35,6 +36,32 @@ const RegistrationPending: ModalContent = ({ onClick }) => {
     </>
   )
 }
+
+const RegistrationPendingUnverified: ModalContent = ({ onClick }) => {
+  return (
+    <>
+      <RegistrationPendingHeader />
+      <Serif my={3} size="3t">
+        This auction requires Artsy to verify your identity before bidding.
+        <br />
+        <br />
+        For details about identity verification, please see the
+        <a
+          target="_blank"
+          href="https://www.artsy.net/identity-verification-faq"
```

Comment:
can we scope this to the environment that we're in?  I believe we can use `getENV('APP_URL')`, similar to what we do here:

https://github.com/artsy/reaction/blob/master/src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx#L103

Reply:
just updated to use a relative path.